samples/sensor/bmi270/src/main.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2021 Bosch Sensortec GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/sensor.h>
+#include <stdio.h>
+
+void main(void)
+{
+	const struct device *dev;
+	struct sensor_value acc[3], gyr[3];
+	struct sensor_value full_scale, sampling_freq, oversampling;
+
+	dev = device_get_binding(DT_LABEL(DT_INST(0, bosch_bmi270)));
+	if (dev == NULL) {
+		printf("Could not get %s device\n",
+		       DT_LABEL(DT_INST(0, bosch_bmi270)));
+		return;
+	}
+
+	printf("Device %p name is %s\n", dev, dev->name);
```

Comment:
I guess it does not print anything meaningful? You need to replace dev->name, `printf("Device %p name is %s\n", dev, DT_LABEL(DT_INST(0, bosch_bmi270)));`

Reply:
The effect is the same as previous. Updated anyway.